### ISSUE TITLE
Visualization for Cell Line Unit of Operations - Gantt Chart

### ISSUE THREAD

**Issue by yuliujpl** [Labcas-NIST/Labcas-ui#62]
Implement the Gantt Chart to visualize the cell line unit of operations use the following metadata: 

- EventType
- EventDate
- Duration= ThisPassageHarvestCellCountTime-LastPassageCellCountTime

Add icon to virtual view to allow user to get to the visualization.

![Image](https://user-images.githubusercontent.com/58526347/235501315-d451e269-3ccc-4df7-939b-b465eb76c559.png)
![Image](https://user-images.githubusercontent.com/58526347/235501314-8c6918df-b0f1-401d-916c-8e8161ee719b.png)

**Comment by yuliujpl** [comment]
HI @hoodriverheather  can you take a look at the gantt chart?
https://ddsa-labcas-staging.jpl.nasa.gov/labcas-ui/c/index.html?collection_id=Cell_Line_Provenance

I will start creating the individual ones for virtual views too!

**Comment by hoodriverheather** [comment]
@yuliujpl this is looking good. I might need some help understanding the colors vs shapes.

**Comment by hoodriverheather** [comment]
@yuliujpl i've been playing with this a bit more while testing some of the other issues. I wonder if users would still want the dataset view to come up when they first click on the collection link. Maybe then they can click or toggle between the gantt chart view and the virtual dataset view? Maybe you already have this figured out?
Also, if the dates/times in the metadata aren't converted to date. Could you convert on ingest?

**Comment by hoodriverheather** [comment]
@yuliujpl we talked more with John after you had to leave the call yesterday. The data that we currently have ingested needs some cleaning based on changes they have been making to the metadata. For example, there should always be a start and a stop EventType. There shouldn't be a big gap in dates for a single Expansion unit. Unit expansions can have one or more EventType=Transfer. This is when the "child" cells are created. John is going to figure out the best way to link the parent cells to the child cells. 
So, bottom line, John is going to clean up this data and send us a file with 3 Expansion Units to start with that have clean data throughout the expansion. :) Right now, it's hard to create a visualization with the data we have.
